File: dash_docs/chapters/dash_canvas/examples/canvas_101.py
import dash
import dash_html_components as html
from dash_canvas import DashCanvas

app = dash.Dash(__name__)
app.config.suppress_callback_exceptions = True

app.layout = html.Div([
    html.H5('Press down the left mouse button and draw inside the canvas'),
    DashCanvas(id='canvas_101')
    ])


if __name__ == '__main__':
    app.run_server(debug=True)

File: dash_docs/chapters/dash_canvas/examples/canvas_101.R
library(dash)
library(dashHtmlComponents)
library(dashCanvas)

app <- Dash$new()

app$layout(htmlDiv(list(
  htmlH5("Press down the left mouse button and draw inside the canvas"),
  dashCanvas(id = 'canvas_101')
)))

app$run_server()
  
